### PR TITLE
Fix #32: Error when reading token

### LINKED ISSUE #32: Error: Encrypted content has been tampered with
I got this error when running the program 

capture d ecran 2016-02-22 a 12 21 17

```
➜ ~/www/veldto.com (master) git-labelmaker
? What is your master password? *********
Failed opening archive: Error: Encrypted content has been tampered with
{ id: 'TOKEN',
  err: 'Failed opening archive: Error: Encrypted content has been tampered with' }
```

### PR DESCRIPTION
**Pull request by Jameskmonger**
This should fix the issue. I have changed the npm version to 0.7.1-alpha.

### CODE REVIEW

**Comment by Jameskmonger** [comment]
@himynameisdave could you CR this and then publish it up to npm as version 0.7.1-alpha?

File: bin/modules/fetchToken.js [hunk 1]
```diff
@@ -28,17 +32,19 @@ module.exports = (rememberedToken) => {
...
+            let token = group.getAttribute('token');
+            res(token);
+          })
+          .catch((e)=>{
+            rej(err(e.message));
+          })
         })
         .catch((e)=>{
-          console.error(err(e.message));
+          rej(err(e.message));
```
Comment (himynameisdave) [inline bin/modules/fetchToken.js:47]: Yeah that'll do it. Good catch @Jameskmonger :+1:

**Comment by himynameisdave** [comment]
Closes #32

**Comment by Jameskmonger** [comment]
Cheers @himynameisdave - don't forget to `npm publish` to push out the latest version :stuck_out_tongue_closed_eyes:

**Comment by himynameisdave** [comment]
haha already have :+1:

**Pull request merged**